bundles/org.openhab.binding.telegram/src/main/java/org/openhab/binding/telegram/internal/TelegramHandler.java

```diff
@@ -138,10 +140,23 @@ public void initialize() {
         TelegramConfiguration config = getConfigAs(TelegramConfiguration.class);
 
         String botToken = config.getBotToken();
-        chatIds.clear();
+        authorizedSenderChatId.clear();
+        receiverChatId.clear();
+
         for (String chatIdStr : config.getChatIds()) {
             try {
-                chatIds.add(Long.valueOf(chatIdStr));
+                if (chatIdStr.trim().startsWith("<")) {
+                    // inbound only
+                    authorizedSenderChatId.add(Long.valueOf(chatIdStr.substring(1)));
```

Comment:
If you `trim()` earlier it is not a good idea to use `substring(1)` on the not trimmed String. Maybe introduced a local variable for the `trimmedChatId and use it here and in the following lines.

Reply:
Good point.Done.